src/System.Drawing.Common/src/System/Drawing/Printing/PrintingPermission.cs

```diff
@@ -1,219 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I remember that we stubbed other CAS types out instead of removing them to reduce the chance of breaking someone. Should we do that here also?

Reply:
This are defined in System.Security.Permissions already, this file was not used at all here in System.Drawing.

https://github.com/dotnet/corefx/blob/master/src/System.Security.Permissions/src/System/Drawing/Printing/PrintingPermission.cs

Reply:
I see. To me that sounds like our Mono colleagues were using it probably. Have you checked with them that removing these files don't break them?

Reply:
I wouldn't think they would use them for System.Drawing because they are part of System.Security.Permissions so they should have them in that assembly and this files are not stubbed vs the ones in S.Security.Permissions are stubbed.

@marek-safar @akoeplinger are you guys currently using this Permission files under System.Drawing?

Reply:
Looks like they are using them: https://github.com/mono/mono/blob/master/mcs/class/System.Drawing/System.Drawing-net_4_x.csproj#L177-L179

So I guess I won't delete this files then.

Reply:
@weshaggard we should find a name convention for this files, because it is confusing to have files that are not used by our projects, they seem to be dead code. Maybe adding a .mono suffix?

Reply:
The best solution (for us) would be to move them to Mono...

Reply:
I agree with that.

Reply:
Given that we don't care about CAS I'd assume we could also just use the stubs from https://github.com/dotnet/corefx/blob/master/src/System.Security.Permissions/src/System/Drawing/Printing/PrintingPermission.cs.

@marek-safar do you agree?

Reply:
@safern using `.Mono.cs` suffix is the existing convention we are already using elsewhere in corefx repo. I'd prefer not to delete the files but adding the suffix instead as the distant goal is to have Mono build profile inside corefx repo.

Reply:
But in this case to not have duplicate files since we don’t care about CAS why not use the files that are present under System.Security.Permissions which are stubbed types?

What is the action required here to update the file names and don’t break you guys?

Reply:
No need to duplicate anything, if you want to make it more clear the files are used by Mono only just add the suffix and we'll update our build scripts

Reply:
What @safern meant is that currently we already duplicated these files as they exist in two places in corefx. We would want to remove them here so that they only exist in S.Security.Permissions.

Reply:
yes, that's fine to keep just single file copy

Reply:
Yes lets keep these deleted and have mono start consuming https://github.com/dotnet/corefx/blob/master/src/System.Security.Permissions/src/System/Drawing/Printing/PrintingPermission.cs instead.

Reply:
Perfect will do that. Thanks :)

Reply:
I updated the PR to remove these files from System.Drawing.Common and you guys will now need to pick them up from System.Security.Permissions. These are the files to pick up:

* https://github.com/dotnet/corefx/blob/master/src/System.Security.Permissions/src/System/Drawing/Printing/PrintingPermission.cs
* https://github.com/dotnet/corefx/blob/master/src/System.Security.Permissions/src/System/Drawing/Printing/PrintingPermissionLevel.cs
* https://github.com/dotnet/corefx/blob/master/src/System.Drawing.Common/src/System/Drawing/Printing/PrintingPermissionAttribute.cs